spec/system/claims/support/claims/download_claims_csv_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe "Download claims CSV", type: :system, service: :claims do
+  let(:support_user) { create(:claims_support_user) }
+
+  before do
+    create_claim
+    user_exists_in_dfe_sign_in(user: support_user)
+    given_i_sign_in
+  end
+
+  scenario "Support user visits the claims index page and downloads CSV" do
+    when_i_visit_claim_index_page
+    then_i_click_call_to_action
+    check_correct_path_called
+  end
```

Comment:
This isn't strictly correct usage of [Given/When/Then](https://www.agilealliance.org/glossary/given-when-then/) syntax.

I think something like this could be a better fit for what you're trying to prove with this test:

```suggestion
  scenario "Support user visits the claims index page and downloads CSV" do
    given_i_am_on_the_claims_index_page
    when_i_click_download_claims
    then_i_receive_a_csv_file
  end
```

Plus I think the test might be more valuable if we check that a CSV file is sent back to the user. It would prove that the endpoint actually produces a CSV, rather than just checking the page path:

```ruby
def then_i_receive_a_csv_file
  expect(response_headers["Content-Type"]).to eq "text/csv"
end
```